internal/pkg/config/filterType.go

```diff
@@ -0,0 +1,56 @@
+package config
+
+import "fmt"
+
+type FilterType int
+
+const (
+	loadBalancer FilterType = iota
```

Comment:
Perhaps name the existing type to kubernetes Service instead of load balancer?

Reply:
Convert these consts to uppercase please @fahad-rauf

Reply:
If I use Uppercase, which I was using before, then I cannot use camel case like @rasheedamir commented earlier to do